Standards/scs-XXXX-vN-key-manager-standard.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Key-Manager Standard
+type: Standard
+status: Draft
+track: IaaS
+---
+
+## Introduction
+
+To encrypt user data like volumes or in the future also Images and ephemeral storage for VMs, the key is need to be known in the infrastructure.
+To provide the key to those operations without including the user every time a Key Manager within the infrastructure can be utilized to store the keys and apply authorization policies on requests.
+OpenStack offers a Key Manager implementation that is named Barbican, which provides these features.
+This standard aims to provide a base level of security for Cloud Service Providers that integrate a Key Manager into their deployments.
+
+## Terminology
+
+| Term | Meaning |
+|---|---|
+| API | Application Programming Interface, often referring to the REST API interfaces provided by OpenStack and related services |
+| Barbican | The Key Manager implementation in OpenStack |
+| CSP | Cloud Service Provider, provider managing the OpenStack infrastructure |
+| IaaS | Infrastructure-as-a-Service |
+| HSM | Hardware Security Module |
+| KEK | Key Encryption Key |
+
+## Motivation
+
+User data encryption requires an encryption key to be known during encryption and decryption processes.
+Key Managers like Barbican provide this functionality on the IaaS-Level.
+Not every IaaS deployment currently offers user data encryption as part of their standard offering.
+A first step towards more security is to encourage CSPs to provide better data security by offering data encryption to the customers.
```

Comment:
"[snip] step towards more security is to encourage [snip] to provide better data security [snip]". The sentence is really awkward.